cli/src/main/java/fr/pilato/elasticsearch/crawler/fs/cli/FsCrawlerCli.java

```diff
@@ -229,6 +224,7 @@ public static void main(String[] args) throws Exception {
                 fsSettings = FsSettings.builder(commands.jobName.get(0))
                         .setFs(Fs.DEFAULT)
                         .setElasticsearch(Elasticsearch.DEFAULT())
+                        .setRest(Rest.DEFAULT)
```

Comment:
I don't think this is needed. We are setting that elsewhere (in `FsCrawlerValidator`) if FsCrawler is started with the rest mode.